src/tenant/tenant.resolver.ts

```diff
@@ -45,4 +46,35 @@ export default class TenantResolver {
 
     return tenant;
   }
+
+  @Mutation(() => Boolean)
+  async markRentAsLate(@Arg('id', () => Int) id: number): Promise<boolean> {
+    const t = await Tenant.findOne({where: {id}});
+    if (t) {
+      const l = await Lease.find({where: {tenantId: id}});
+      if (l.length === 0) {
+        throw new Error("Tenant doesn't have any leases");
+      }
+
+      t.isRentLate = true;
+      await t.save();
+    }
+
+    return true;
+  }
+
+  @Mutation(() => Boolean)
+  async unmarkRentAsLate(@Arg('id', () => Int) id: number): Promise<boolean> {
+    const t = await Tenant.findOne({where: {id}});
+    if (t) {
+      const l = await Lease.find({where: {tenantId: id}});
+      if (l.length === 0) {
+        throw new Error("Tenant doesn't have any leases");
+      }
+      t.isRentLate = false;
+      await t.save();
+    }
```

Comment:
We could fetch the tenant only when we're sure to perform the update. Same suggestion applies to the previous method.

```suggestion
    const lease = await Lease.find({where: {tenantId: id}});
    if (lease.length === 0) { /* */ }
    const tenant = await Tenant.findOne({where: {id}});
    if (tenant) {
        tenant.isRentLate = false;
         await tenant.save();
    }
```